### ISSUE TITLE
s390x is failing new tests scal.c

### ISSUE THREAD

**Issue by mattip** [xianyi/OpenBLAS#4828]
I am building OpenBLAS [on s390x](https://app.travis-ci.com/github/MacPython/openblas-libs/jobs/624579284) on travis inside a `quay.io/pypa/manylinux2014_s390x` docker (which uses gcc 10.4) via
```
CFLAGS="-fvisibility=protected -Wno-uninitialized" \
make BUFFERSIZE=20 DYNAMIC_ARCH=1 USE_OPENMP=0 NUM_THREADS=64 \
    BINARY=64 INTERFACE64=1 SYMBOLSUFFIX=64_ LIBNAMESUFFIX=64_ \
    OBJCONV=$PWD/objconv/objconv
```

using commit d11e7340 and am seeing some
<details><summary>test failures</summary>

```
TEST 39/107 zscal:i_0inf_inc_2 [FAIL]

  ERR: test_zscal.c:586  should be true

TEST 40/107 zscal:i_0inf [FAIL]

  ERR: test_zscal.c:574  should be true

TEST 41/107 zscal:inf_i_inc_2 [OK]

TEST 42/107 zscal:inf_i [OK]

TEST 43/107 zscal:i_inf_inc_2 [OK]

TEST 44/107 zscal:i_inf [OK]

TEST 45/107 zscal:nan_i_inc_2 [OK]

TEST 46/107 zscal:nan_i [OK]

TEST 47/107 zscal:i_nan_inc_2 [OK]

TEST 48/107 zscal:i_nan [OK]

TEST 49/107 cscal:i_0inf_inc_2 [FAIL]

  ERR: test_zscal.c:440  should be true

TEST 50/107 cscal:i_0inf [FAIL]

  ERR: test_zscal.c:426  should be true

TEST 51/107 cscal:inf_i_inc_2 [OK]

TEST 52/107 cscal:inf_i [OK]

TEST 53/107 cscal:i_inf_inc_2 [FAIL]

  ERR: test_zscal.c:385  should be true

TEST 54/107 cscal:i_inf [FAIL]

  ERR: test_zscal.c:371  should be true

TEST 55/107 cscal:nan_i_inc_2 [OK]

TEST 56/107 cscal:nan_i [OK]

TEST 57/107 cscal:i_nan_inc_2 [FAIL]

  ERR: test_zscal.c:331  should be true

TEST 58/107 cscal:i_nan [FAIL]

  ERR: test_zscal.c:317  should be true

TEST 59/107 dscal:inf_nan_inc_2 [OK]

TEST 60/107 dscal:inf_nan [OK]

TEST 61/107 dscal:nan_inf_inc_2 [OK]

TEST 62/107 dscal:nan_inf [OK]

TEST 63/107 dscal:inf_0_inc_2 [OK]

TEST 64/107 dscal:inf_0 [OK]

TEST 65/107 dscal:0_inf_inc_2 [FAIL]

  ERR: test_zscal.c:226  should be true

TEST 66/107 dscal:0_inf [FAIL]

  ERR: test_zscal.c:214  should be true

TEST 67/107 dscal:nan_0_inc_2 [OK]

TEST 68/107 dscal:nan_0 [OK]

TEST 69/107 dscal:0_nan_inc_2 [FAIL]

  ERR: test_zscal.c:180  should be true

TEST 70/107 dscal:0_nan [FAIL]

  ERR: test_zscal.c:168  should be true

TEST 71/107 sscal:inf_nan_inc_2 [OK]

TEST 72/107 sscal:inf_nan [OK]

TEST 73/107 sscal:nan_inf_inc_2 [OK]

TEST 74/107 sscal:nan_inf [OK]

TEST 75/107 sscal:inf_0_inc_2 [OK]

TEST 76/107 sscal:inf_0 [OK]

TEST 77/107 sscal:0_inf_inc_2 [FAIL]

  ERR: test_zscal.c:77  should be true

TEST 78/107 sscal:0_inf [FAIL]

  ERR: test_zscal.c:65  should be true

TEST 79/107 sscal:nan_0_inc_2 [OK]

TEST 80/107 sscal:nan_0 [OK]

TEST 81/107 sscal:0_nan_inc_2 [FAIL]

  ERR: test_zscal.c:31  should be true

TEST 82/107 sscal:0_nan [FAIL]

  ERR: test_zscal.c:19  should be true
```

</details>

Note I am not specifying `TARGET` on s390x since I have no idea what is an acceptable "minimum". 

This is not a deal breaker for NumPy/SciPy since we do not release wheels for that platform, but I thought I would report it anyway.

**Comment by martin-frbg** [comment]
somewhat expected but I had had problems accessing our Jenkins CI for a while (one needs to be logged in there to see the logs). 
"acceptable minimum" for dynamic_arch on zarch would be the zarch_generic target I guess, but I have no idea if there is anything of practical importance left below the Z13

**Comment by martin-frbg** [comment]
Closing as fixed (verified by #4843)

**Comment by mattip** [comment]
Thanks!
